pyHS100/cli.py

```diff
@@ -72,14 +81,28 @@ def discover(ctx, timeout, discover_only):
     click.echo("Discovering devices for %s seconds" % timeout)
     found_devs = Discover.discover(timeout=timeout).items()
     if not discover_only:
-        for ip, dev in found_devs:
+        for _, dev in found_devs:
```

Comment:
I think it's better to leave it as it was, as it makes it easy for a reader to see what it is behind the variable. So either revert this change, or use `found_devs.items()` and drop it altogether.